cg/apps/lims/api.py

```diff
@@ -395,30 +396,38 @@ def get_sample_project(self, sample_id: str) -> str:
         """Get the lims-id for the project of the sample"""
```

Comment:
Update docstring and return type hint to account when the sample does not exist. Same for method above
```suggestion
        """Get the lims-id for the project of the sample, if the sample exists."""
```

Reply:
Good catch ⭐ 

[31093b1](https://github.com/Clinical-Genomics/cg/pull/3153/commits/31093b1d65805f780abe76b89b07a07b90a3679c)